jobs/config.json

```diff
@@ -7797,6 +7797,22 @@
       "sig-aws"
     ]
   },
+  "ci-kubernetes-e2e-kops-gce": {
+    "args": [
+      "--cluster=e2e-kops-gce.k8s.local",
+      "--env-file=jobs/platform/kops_gce.env",
+      "--env-file=jobs/env/ci-kubernetes-e2e-kops-gce.env",
```

Comment:
bikeshed: You can use --env=KOPS_PUBLISH_GREEN_PATH=gs://kops-ci/bin/latest-ci-gce-green.txt and drop the env file. Later on this probably should be some flag in kubetest?

We are moving towards get rid of env and env files.